problems/4.arrays.js

```diff
@@ -2,28 +2,32 @@
 * @param {number[]} arr - The input array
 * @returns {number} - The average of the first and last values
 */
-function firstAndLastAverage() {
-
+function firstAndLastAverage(arr) {
+  return (arr[0] + arr.length -1) / 2
+  
 }
+console.log(firstAndLastAverage[2,3,4,5,6])
 
 /** Duplicates all values inside an array.
 * Sample input: [1,4,2,5]
 * Sample output: [1,4,2,5,1,4,2,5]
 * @param {number[]} arr - The input array
 * @returns {number[]} - The duplicated array
 */
-function duplicateArray() {
-
+function duplicateArray(arr) {
+  return arr.push(arr + arr) 
 }
+console.log(duplicateArray([1,4,6,7,8]))
 
 /** Removes the first 3 elements from an array. If the array has fewer than 3 elements, return an empty array.
 * @param {any[]} arr - The input array
 * @returns {any[]} - The array with the first 3 elements removed
 */
-function removeFirstThreeElements() {
-
+function removeFirstThreeElements (arr) {
+   return arr.splice(0,3)
```

Comment:
This would return the three elements removed. You want the arr after this has happened.